grafana_loki_openapi.yaml

```diff
@@ -46,6 +46,56 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/QueryResponseBody"
+  /loki/api/v1/query_range:
+    get:
+      tags:
+        - Query Range
+      parameters:
+        - name: query
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: limit
+          in: query
+          schema:
+            type: integer
+            minimum: 1
+            default: 100
+        - name: start
+          in: query
+          schema:
+            type: integer
```

Comment:
Do we want to limit `start`, `end` and `step` to integer only or allow other formats as per documentation https://grafana.com/docs/loki/latest/api/#timestamp-formats?

Reply:
Done - I created a schema for Timestamp fields so that it can be reused for multiple fields in multiple endpoints